tests/e2e/pageobjects/ide/GitPlugin.ts

```diff
@@ -75,12 +83,19 @@ export class GitPlugin {
         await this.driverHelper.waitPresence(By.xpath(`//div[text()='Staged Changes']/parent::div/parent::div/parent::div/following-sibling::div//span[text()=\'${expectedStagedItem}\']`), timeout);
     }
 
-    async commitFromScmView(timeout: number = TimeoutConstants.TS_GIT_CONAINER_INTERACTION_TIMEOUT) {
+    async commitFromCommandMenu(timeout: number = TimeoutConstants.TS_GIT_CONAINER_INTERACTION_TIMEOUT) {
         Logger.debug('GitPlugin.commitFromScmView');
+        await this.topMenu.selectOption('View', 'Find Command...');
+        await this.quickOpenContainer.typeAndSelectSuggestion('Commit', 'Git: Commit');
+    }
 
-        await this.driverHelper.waitAndClick(By.id('__scm-view-container_title:__plugin.scm.title.action.git.commit'), timeout);
+    async pushChangesFromCommandMenu(timeout: number = TimeoutConstants.TS_GIT_CONAINER_INTERACTION_TIMEOUT) {
```

Comment:
The same https://github.com/eclipse/che/pull/18374/files#r523976305

Reply:
The timeout has been removed